composer.lock

```diff
@@ -5815,37 +5981,39 @@
                     "type": "tidelift"
                 }
             ],
-            "time": "2020-07-05T13:08:05+00:00"
+            "time": "2020-10-27T19:25:29+00:00"
         },
         {
             "name": "vimeo/psalm",
-            "version": "3.12.2",
+            "version": "3.18.2",
```

Comment:
Please do a `git revert` on `composer.lock`, then try a `composer update phpunit/phpunit` to isolate out this `vimeo/psalm` upgrade.

`vimeo/psalm` is correct here (in the issues reported in this patch), but that's my problem to solve later :-)